sdv/multi_table/hma.py

```diff
@@ -6,6 +6,7 @@
 
 import numpy as np
 import pandas as pd
+from packaging import version
```

Comment:
can we avoid using packaging? It's not listed as an sdv dependency and is probably installed by one of the other libraries. It seems like matplotlib and plotly require it and those are also not direct dependencies for SDV